### PR TITLE
Add my name to About Me

### PR DESCRIPTION
**Pull request by bernars-usa**
This is my name. I like it.

cc @sharmaneil for review please! 🐃

### CODE REVIEW

**Comment by sharmaneil** [comment]
I thought your name is sheri?

File: about-me.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ![Picture of me]()
 
-Hi, I'm ________. I'm a ________ at *__________*.
+Hi, I'm Charlene Bernard. I'm a ________ at *__________*.
```
Comment (sharmaneil) [inline about-me.md:5]: i thought your name is sheri??? WHO ARE YOU?

Comment (bernars-usa) [reply about-me.md:5]: My legal name is Charlene. But only my mom calls me that, when I'm in trouble! Also on government forms. 🤷‍♀

Comment (sharmaneil) [reply about-me.md:5]: Ah makes sense. Cool cool!

**Comment by sharmaneil** [comment]
🚢 Looks good Charlene!

**Pull request merged**